senza/components/taupage_auto_scaling_group.py

```diff
@@ -1,33 +1,42 @@
+import json
+import re
+import sys
+import textwrap
 
 import click
 import pierone.api
-import textwrap
 import yaml
-import json
-import sys
-import re
-
 from senza.aws import resolve_referenced_resource
 from senza.components.auto_scaling_group import component_auto_scaling_group
 from senza.docker import docker_image_exists
 from senza.utils import ensure_keys
 
-
 _AWS_FN_RE = re.compile(r"('[{]{2} (.*?) [}]{2}')", re.DOTALL)
 
 
 def check_docker_image_exists(docker_image: pierone.api.DockerImage):
     if 'pierone' in docker_image.registry:
         try:
             exists = pierone.api.image_exists('pierone', docker_image)
+            if exists:
+                image_tag = pierone.api.get_image_tag('pierone', docker_image)
+                if image_tag['severity_fix_available'] not in ['NOT_PROCESSED_YET',
+                                                               'COULDNT_FIGURE_OUT',
+                                                               'NO_CVES_FOUND',
+                                                               'TOO_OLD']:
```

Comment:
@sarnowski Could you review that those stuatuses are the ones we do **not** want to show the `warning`? I am not sure about the `TOO_OLD` and `NOT_PROCESSED_YET` maybe we want to show the warning for those statuses too.